app/models/user.rb

```diff
@@ -15,4 +15,10 @@ class User < ApplicationRecord
 
   mount_uploader :image_url, ImageUploader
   mount_uploader :resume, AttachmentUploader
+
+  enum status: ["active", "inactive"]
+
+  scope :default, -> {where(role: "default")}
```

Comment:
@katiekeel Nice work with the scopes! For a moment I thought that the enum helper method .default and scoping User is redundant, but I see that the scope gives us a class method and the other is an instance method.